build/clifm/build.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+
+# Copyright 2025 OmniOS Community Edition (OmniOSce) Association.
+
+. ../../lib/build.sh
+
+PROG=clifm
+VER=1.26.3
+PKG=ooce/application/clifm
+SUMMARY="$PROG"
+DESC="A shell-like, text-based terminal file manager"
+
+set_arch 64
+set_clangver
+
+#SKIP_SSP_CHECK=1
```

Comment:
if the SSP check does not need to be overridden, please remove it instead of just commenting it